RELEASE.MD

```diff
@@ -0,0 +1,28 @@
+# Releasing new Matej PHP Client version
+
+1. Check [open pull-requests](https://github.com/lmc-eu/matej-client-php/pulls), if there are any, you may coordinate releasing of a new version with authors of these pull requests.
```

Comment:
why are there only `1.` ?

Reply:
Every time I use this markdown numbering, someone asks this question :-D. It seems like not so know markdown feature 🤔.

Anyway you can check the markdown result here:  https://github.com/lmc-eu/matej-client-php/blob/30ee967130058c17d89bf0fd60cba3809d186285/RELEASE.MD